src/dbnode/storage/namespace/options_test.go

```diff
@@ -53,7 +53,7 @@ func TestOptionsEqualsIndexOpts(t *testing.T) {
 
 func TestOptionsEqualsSchema(t *testing.T) {
 	o1 := NewOptions()
-	s1, err := LoadSchemaRegistry(GenTestSchemaOptions())
+	s1, err := LoadSchemaRegistry(GenTestSchemaOptions("schematest"))
```

Comment:
is this string referring to a specific file that is being loaded? Should we make it a constant?

Reply:
bump

Reply:
This is the relative path (comparing to current test file) to the location of the test proto file. 
Since I want to reuse this in storage test as well, I make this an input, in storage test, it will use "namespace/schematest".